backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/error/AdIssueFactory.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * #%L
+ * de.metas.adempiere.adempiere.base
+ * %%
+ * Copyright (C) 2022 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.error;
+
+import de.metas.util.Check;
+import de.metas.util.Services;
+import de.metas.util.StringUtils;
+import lombok.experimental.UtilityClass;
+import org.adempiere.ad.service.ISystemBL;
+import org.adempiere.util.net.NetUtils;
+import org.compiere.Adempiere;
+import org.compiere.model.I_AD_Issue;
+import org.compiere.model.I_AD_System;
+import org.compiere.util.DB;
+
+import java.util.Properties;
+
+import static org.adempiere.model.InterfaceWrapperHelper.newInstance;
+
+@UtilityClass
+public class AdIssueFactory
+{
+	public I_AD_Issue createNewIssueRecord(final Properties ctx)
+	{
+		final I_AD_Issue issue = newInstance(I_AD_Issue.class);
+
+		final ISystemBL systemBL = Services.get(ISystemBL.class);
+		final I_AD_System system = systemBL.get(ctx);
```

Comment:
why not using 
```
			final ADSystemInfo adSystem = Services.get(ISystemBL.class).get();
```

and more the columns that are missing (because we are not using them, not maintaining them, simply just set some crap values for now, e.g.
```
issue.setUserName("-");
```